ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -164,7 +167,9 @@ export default function CollectiblesItems({
                 variant={TYPOGRAPHY.H5}
                 margin={[0, 0, 0, 2]}
               >
-                {`${collectionName} (${collectibles.length})`}
+                {`${collectionName ?? t('unknownCollection')} (${
```

Comment:
I'm using `Unknown Collection` as fallback when a `collectionName` either does not exist or is not fetched correctly. @rachelcope @SayaGT Would you prefer a different fallback here?

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/15034/commits/0db81311e9ff31b96f69750747e46e9c8bc09958

<img width="776" alt="Screen Shot 2022-06-29 at 12 11 57 PM" src="https://user-images.githubusercontent.com/8112138/176545501-3765e873-1ec7-481d-aa17-880a75112cc9.png">